plugins/modules/ec2_scaling_policy.py

```diff
@@ -303,7 +479,6 @@ def create_scaling_policy(connection, module):
         policy = policies[0]
         for key in params:
             if params[key] != policy.get(key):
-                changed = True
```

Comment:
We should still need this `changed` var to be set here.

Reply:
I think we're good to go here: https://github.com/ansible-collections/community.aws/blob/b5a7e2388488d347d6d6ff8af6e23b88fcb5af69/plugins/modules/ec2_scaling_policy.py#L480